src/go/guestagent/pkg/tracker/portstorage.go

```diff
@@ -79,9 +80,7 @@ func (p *portStorage) getAll() map[string]nat.PortMap {
 func copyPortMap(m nat.PortMap) nat.PortMap {
 	portMap := make(nat.PortMap, len(m))
 
-	for k, v := range m {
-		portMap[k] = v
-	}
+	maps.Copy(portMap, m)
 
 	return portMap
```

Comment:
```suggestion
	return maps.Clone(m)
```

Or even better, fix the callers to use `maps.Clone`.